astropy/coordinates/tests/test_regression.py

```diff
@@ -538,23 +537,6 @@ def test_regression_6446():
     assert sio1.getvalue() == sio2.getvalue()
 
 
-def test_regression_6448():
-    """
-    This tests the more narrow problem reported in 6446 that 6448 is meant to
-    fix. `test_regression_6446` also covers this, but this test is provided
-    so that this is still tested even if YAML isn't installed.
-    """
-    sc1 = SkyCoord([1, 2], [3, 4], unit='deg')
-    # this should always succeed even prior to 6448
-    assert sc1.galcen_v_sun is None
-
-    c1 = SkyCoord(1, 3, unit='deg')
-    c2 = SkyCoord(2, 4, unit='deg')
-    sc2 = SkyCoord([c1, c2])
-    # without 6448 this fails
-    assert sc2.galcen_v_sun is None
-
-
```

Comment:
:+1: Was confused about why this was being removed at first, but it makes sense.  Very astute!

Reply:
Thanks!